apps/ideaspace/src/components/modules/WorkshoppingPage/WorkshoppingPage.js

```diff
@@ -92,4 +124,4 @@ function WorkshoppingPage() {
   }
 }
 
-export default WorkshoppingPage;
+// export default WorkshoppingPage;
```

Comment:
if we are adding the export statement to the same line as where we define the function, let's just delete this line altogether instead of commenting it out